composer.json

```diff
@@ -108,7 +112,7 @@
         ],
         "coding-standards-check": [
             "@coding-standards-check/phpcs",
-            "@coding-standards-check/drupal-check",
+            "@code-analysis",
```

Comment:
We should remove “code analysis” from “coding standards check” and call “code analysis” when needed.

Reply:
Removed